Editing CLO section of Infra MachineSet topc

mburke5678 opened the pull request:
Updating the section on creating an Infra machineset to add how to move cluster logging to an Infra node.

kalexand-rh commented on modules/infrastructure-moving-logging.adoc:
```diff
@@ -3,10 +3,84 @@
+You should set your MachineSet to use at least 6 replicas.
 ====
+
+.Prerequisite
```
Prerequisites

kalexand-rh commented on modules/infrastructure-moving-logging.adoc:
```diff
@@ -3,10 +3,84 @@
+
+.Prerequisite
+
+* Cluster logging and Elasticsearch must be installed. These features are not installed by default.
```
Logging

mburke5678 replied:
We have been using `cluster logging` in lowercase when talking about the feature in general.

kalexand-rh commented on modules/infrastructure-moving-logging.adoc:
```diff
@@ -3,10 +3,84 @@
+....
+
+----
+<1> Add a `nodeSelector` parameter with the appropriate `<key>: <value>` pair to the component you want to move.
```
Will you add an example that references the `node-role.kubernetes.io/infra: ""` `nodeSelector`?

mburke5678 replied:
Is the `node-role.kubernetes.io/infra: ""` node selector required to your knowledge? I couldn't get it to work with that format. I tried the simple `tag=value` format, which was how QE tested the feature, and it worked.

kalexand-rh replied:
If it's not, then we should check the other node selectors.

mburke5678 commented:
@kalexand-rh I will have QE take a look. Wanted to make sure the material fit into the topic properly first. Thanks for looking!

kalexand-rh commented:
@mburke5678, I'm happy to do it! If you add the example that I asked for, I think it'll fit in just fine.

mburke5678 commented:
@anpingli Can you please take a look? 
Also, @kalexand-rh [suggested I should make the node selectors in the custom resource `node-role.kubernetes.io/infra: ""`](https://github.com/openshift/openshift-docs/pull/15600#discussion_r298237466).
When I changed them from `es: yes` to `node-role.kubernetes.io/infra: ""` I got 6 new fluentd pods on the right node (the original 6 Fluentd pods remain, even after I remove the `es: yes` selector from the nodes). 
The Elasticsearch pods did not move {even after I remove the `es: yes` selector from the nodes)/
The Kibana pod moved the new nodes.

anpingli commented:
@mburke5678  I think the goal is to place the ES pods into specified nodes, as ES pods require huge CPU/Memory/Disk resources.    so it is suggested adding nodeSelector for ES.   it is optional to add nodeSelector for kibana/cronjob.   It is not suggested to add nodeSeletor for damonset, as we expected the fluentd/rsyslog locate in each node

mburke5678 commented:
@anpingli I removed the node selector from the `spec.collector.logs.fuentd` section. And added a short note at the top "you can move the Elasticsearch pods to a separate node because of 
high CPU, memory, and disk requirements."

anpingli commented:
LGTM

mburke5678 commented:
@ewolinetz Please take a look at this procedure to move the cluster logging components to new nodes.

ewolinetz commented on modules/infrastructure-moving-logging.adoc:
```diff
@@ -3,10 +3,85 @@
+
+* Cluster logging and Elasticsearch must be installed. These features are not installed by default.
+
+* If needed, get the name of the Cluster Logging Custom Resource in the openshift-logging project:
```
the clusterlogging CR will always need to be named `instance` fyi. if it isn't the operator will print out a message and not take action on it.

ewolinetz commented:
small point about the name of the CLO watched resource. Non-blocker though, lgtm.

mburke5678 commented:
/cherrypick enterprise-4.2
